qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/modules/kubernetes/services/prefect/chart/templates/prefect.yaml

```diff
@@ -38,19 +42,21 @@ spec:
             - name: NAMESPACE
               value: {{ .Values.namespace }}
             - name: IMAGE_PULL_SECRETS
-              value: ''
+              value: {{ .Values.prefect_agent.job.imagePullSecrets | toJson | quote }}
             - name: PREFECT__CLOUD__AGENT__LABELS
-              value: '[]'
+              value: {{ .Values.prefect_agent.prefectLabels | toJson | quote }}
+            - name: PREFECT__CONTEXT__SECRETS__GCP_CREDENTIALS
+              value: {{ .Values.prefect_agent.context.secrets.gcp_credentials | toJson | quote }}
             - name: JOB_MEM_REQUEST
-              value: ''
+              value: {{ .Values.prefect_agent.job.resources.requests.memory | quote }}
             - name: JOB_MEM_LIMIT
-              value: ''
+              value: {{ .Values.prefect_agent.job.resources.limits.memory | quote }}
             - name: JOB_CPU_REQUEST
-              value: ''
+              value: {{ .Values.prefect_agent.job.resources.requests.cpu | quote }}
             - name: JOB_CPU_LIMIT
-              value: ''
+              value: {{ .Values.prefect_agent.job.resources.limits.cpu | quote }}
             - name: IMAGE_PULL_POLICY
-              value: ''
+              value: {{ .Values.prefect_agent.job.imagePullPolicy | quote }}
```

Comment:
Does setting these environment variables actually do anything?  Is this a prefect thing?  If so, can you point me to the documentation describing them please?  Thanks!

Reply:
Sure, [this one](https://docs.prefect.io/orchestration/agents/kubernetes.html) exaplains the Agent role in Prefect and [this one](https://docs.prefect.io/orchestration/agents/overview.html#environment-variables) talks about the environment variables usage

Reply:
Thanks, I can't find for example, JOB_MEM_REQUEST anywhere in their documentation.  How did you find about that env var in particular?